server/app/services/languageProcessor/Processor.java

```diff
@@ -52,6 +57,13 @@
       questionMapping[1] = "NoIdFound";
     }
 
+    questionService.addQuestion(question);
+
+    if(question.contains("list all questions")){
+      questionMapping[0] = "getQuestions";
```

Comment:
Can you please add comment to explain what is a `questionMapping`? And why you assign "getQuestions" for both `[0] [1]`?